pkg/config/mapper.go

```diff
@@ -0,0 +1,98 @@
+package config
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/logandavies181/arnlike"
+)
+
+// SSOArnLike returns a string that can be passed to arnlike.ArnLike to
+// match canonicalized IAM Role ARNs against. Assumes Validate() has been called.
+func (m *RoleMapping) SSOArnLike() string {
+	if m.SSO == nil {
+		return ""
+	}
+
+	var partition string
+	if m.SSO.Partition == "" {
+		partition = "aws"
+	}
+
+	return strings.ToLower(fmt.Sprintf("arn:%s:iam::%s:role/AWSReservedSSO_%s_*", partition, m.SSO.AccountID, m.SSO.PermissionSetName))
```

Comment:
what would happen if a k8s admin set `sso.accountID` or `sso.permissionSetName` to `*`?